pkg/api/api.go

```diff
@@ -251,6 +251,9 @@ func (hs *HTTPServer) registerRoutes() {
 		apiRoute.Get("/plugins", Wrap(hs.GetPluginList))
 		apiRoute.Get("/plugins/:pluginId/settings", Wrap(GetPluginSettingByID))
 		apiRoute.Get("/plugins/:pluginId/markdown/:name", Wrap(GetPluginMarkdown))
+		apiRoute.Get("/plugins/:pluginId/health", Wrap(hs.CheckHealth))
+		apiRoute.Any("/plugins/:pluginId/resources", Wrap(hs.CallResource))
+		apiRoute.Any("/plugins/:pluginId/resources/*", Wrap(hs.CallResource))
```

Comment:
should this be "resources" plural or "resource"?  

I am not particularly wild about exposing the word "resource" in the API, but it does not really matter... maybe `/plugins/:pluginId/res/*`?

Reply:
We can discuss that later. Want to get this merged for now. Will open an issue